qiskit_experiments/library/randomized_benchmarking/layer_fidelity_unitary.py

```diff
@@ -0,0 +1,606 @@
+                If this is set to false the code runs
+                simultaneous direct 1+2Q RB without a barrier across all qubits.
+            min_delay (List[int]): Optional. Define a minimum delay in each 2Q layer in units of dt. This
+                delay operation will be applied in any 1Q edge of the layer during the 2Q gate layer
+                in order to enforce a minimum duration of the 2Q layer. This enables some crosstalk
+                testing by removing a gate from the layer without changing the layer duration. If not
+                None then is a list equal in length to the number of two_qubit_layers.  Note that
+                this options requires at least one 1Q edge (a qubit in physical_qubits but
+                not in two_qubit_layers) to be applied. Also will not have an impact on the 2Q gates
+                if layer_barrier=False.
+        """
+        options = super()._default_experiment_options()
+        options.update_options(
+            lengths=None,
+            num_samples=None,
+            seed=None,
+            two_qubit_layers=None,
+            two_qubit_gates=None,
+            two_qubit_basis_gates=None,
+            one_qubit_basis_gates=None,
+            clifford_synthesis_method=DEFAULT_SYNTHESIS_METHOD,
+            layer_barrier=True,
+            min_delay=None,
+        )
+        return options
+
+    def set_experiment_options(self, **fields):
+        """Set the experiment options.
+
+        Args:
+            fields: The fields to update the options
+
+        Raises:
+            AttributeError: If the field passed in is not a supported options
+        """
+        for field in fields:
+            if field in {"two_qubit_layers"}:
+                if (
+                    hasattr(self._experiment_options, field)
+                    and self._experiment_options[field] is not None
+                ):
+                    raise AttributeError(f"Options field {field} is not allowed to update.")
+        super().set_experiment_options(**fields)
+
+    @classmethod
+    def _default_transpile_options(cls) -> Options:
+        """Default transpiler options for transpiling RB circuits."""
+        return Options(optimization_level=1)
+
+    def set_transpile_options(self, **fields):
+        """Transpile options is not supported for LayerFidelity experiments.
+
+        Raises:
+            QiskitError: If `set_transpile_options` is called.
+        """
+        raise QiskitError(
+            "Custom transpile options is not supported for LayerFidelity experiments."
+        )
+
+    def _set_backend(self, backend: Backend):
+        """Set the backend V2 for RB experiments since RB experiments only support BackendV2."""
+        if BackendV1 is not None and isinstance(backend, BackendV1):
+            super()._set_backend(BackendV2Converter(backend, add_delay=True))
+        else:
+            super()._set_backend(backend)
```

Comment:
This is a new experiment. Let's start off not supporting BackendV1.

```suggestion
        super()._set_backend(backend)
```